### ISSUE TITLE
messagebus service related error

### ISSUE THREAD

**Issue by m4ge** [sgnl05/sgnl05-sssd#29]
When provisioning a CentOS 7.2 vagrant box I get the following error:
```
Error: Could not enable messagebus: 
Error: /Stage[main]/Sssd::Service/Service[messagebus]/ensure: change from stopped to running failed: Could not enable messagebus: 
Warning: /Stage[main]/Sssd::Service/Service[sssd]: Skipping because of failed dependencies
Warning: /Stage[main]/Sssd/Anchor[sssd::end]: Skipping because of failed dependencies
```
Running yum update on the box seem to fix the problem and so does setting
```
service_dependencies = ['dbus']
```
After looking around I have stumbled upon this: https://bugzilla.redhat.com/show_bug.cgi?id=1184960. Could that be the cause of my problem? If so, should the default value for service_dependencies be ['dbus'] rather than ['messagebus']? Is it safe to have it as an empty array?

**Comment by ndelic0** [comment]
@m4ge try setting service_dependencies = [] for RH7 - messagebus daemon is nothing but the symlink to a dbus

**Comment by m4ge** [comment]
@ndelic0 should the default setting for service_dependencies for RHEL7 be []? This would remove the need to specify custom options and the module would work out of the box. I'm happy to submit the patch for that if that is the solution to the issue

**Comment by ndelic0** [comment]
@m4ge service_dependencies is already set as class param - basically module already allows one set this in Hiera.  It would be nice to have it working out of the box though.

**Comment by m4ge** [comment]
@ndelic0 I shall submit a pull request to that accord

**Comment by ndelic0** [comment]
@m4ge thank you, thank you

**Comment by sgnl05** [comment]
Fixed in  0.4.1
